docs/source/getting_started/05_benchmarks.rst

```diff
@@ -2,12 +2,72 @@
 CV Model Benchmarks
 *******************
 
-At PeekingDuck, we aim to give users as much flexibility as possible to cater to every possible use case.
+At PeekingDuck, we aim to give users as much flexibility as possible to cater to every possible scenario.
 From real-time object detection to high-accuracy pose estimation, PeekingDuck houses a wide range of models.
 The full list of models can be found :mod:`here <peekingduck.pipeline.nodes.model>`.
 
-We are currently running benchmarks on model performance and FPS, and will be updating this document with our benchmark results.
-We hope that this would allow users to have a quick understanding on the relative performance of each model, thereby enabling
-users to decide on the model most suited for their use case.
+Choosing the right model, given hardware constraints, is usually about making a trade-off between inference speed
+(FPS, or Frames Per Second, in the case of CV), and model performance (mAP, or mean average precision for some CV tasks).
+We would like to use our best performing models all the time, but in the real world, we are often limited by factors such
+as project costs and power/size requirements of the hardware.
 
+We hope that these benchmarks would help you choose the most suitable model for your application.
 
+
+Inference Speed
+===============
+
+Test Hardware
+-------------
+We ran FPS benchmarks on the following devices to compare CPU vs GPU performance:
+ | - MacBook Pro 2017, with 2.9 GHz Quad-Core Intel Core i7 and 16GB RAM
+ | - NVIDIA A100 GPU, paired with 2.2 GHz 6-Core Intel Xeon CPU and 85GB RAM
+
+Test Conditions
+---------------
+The following test conditions were followed:
+ | - ``input.recorded``, the model of interest, and ``dabble.fps`` nodes were used to perform inference on videos
+ | - 2 videos were used to benchmark each model, one with only 1 human (``single``), and the other with multiple humans (``multiple``)
+ | - Both videos are about 1 minute each, recorded at ~30 FPS, which translates to about 1,800 frames to process for each
+ | - 1280x720 (HD ready) resolution was used, as a bridge between 640x480 (VGA) of poorer quality webcams, and 1920x1080 (Full HD) of CCTVs
+ | - All unnecessary processes such as browsers were closed to prevent IO/resource contention
```

Comment:
``` suggestion
All unnecessary processes, such as browsers, were closed to prevent IO/resource contention
```

Reply:
Added commas, but the `|` operator is needed for newlines in .rst files